source/blog/2016-10-04-ovirt-provider-ovn.md

```diff
@@ -0,0 +1,119 @@
+---
+title: oVirt Software Defined Networking, The OVN Network Provider
+author: mmirecki
+date: 2016-10-03 14:21:00 UTC
+tags: community, howto, blog, network
+comments: true
+published: true
+---
+
+oVirt offers not only its own internal networking, but also an API for external network providers. This API enables using external network management software inside environments managed by oVirt and takes advantage of their extended capabilities. 
+One of such solutions is OVN: Open Virtual Network. OVN is an OVS (Open vSwitch) extension that brings Software Defined Networking to OVS.
+
+OVN enables support for virtual networks abstraction by adding native OVS support for virtual L2 and L3 overlays.
+
+READMORE
+
+The oVirt provider for OVN consists of two parts:
+* The oVirt OVN driver
+* The oVirt OVN provider
+
+## oVirt OVN Driver
+
+The oVirt OVN driver is the Virtual Interface Driver placed on oVirt hosts that handles the wiring of VM NICs to OVN networking.
+
+The driver allows vdsm, libvirt, and OVN to interact whenever a NIC is pluged in such a way that the VM NIC is added to an appropriate OVN Logical Switch and the appropriate OVN overlays on all the hosts in the oVirt environment.
```

Comment:
I am not a web designer, but wouldn't line break prevent browser from adjusting the flow to the width of the reader terminal ?